.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
```

Comment:
can we add special treatment for issues marked with long-term?

Reply:
yes, it is configurable with arg `exempt-issue-labels`, here is my test:

- long-term issue: <https://github.com/zhengfeiwang/gh-actions/issues/7>
- not long-term issue: <https://github.com/zhengfeiwang/gh-actions/issues/8>